operator/pkg/validate/common.go

```diff
@@ -180,6 +181,25 @@ func validateIntRange(path util.Path, val interface{}, min, max int64) util.Erro
 	return util.NewErrs(err)
 }
 
+// validateAnalysisInterval checks whether val is valid time duration and greater than 1 second.
+func validateAnalysisInterval(path util.Path, val interface{}) util.Errors {
+	k := reflect.TypeOf(val).Kind()
```

Comment:
I believe type validation should not be necessary as `env.RegisterDurationVar().Get()` already returns a duration.  The only thing we need to check is that the value is greater than 1s, and that can be done in pilot, rather than the operator.

Reply:
will do.